src/settings/toml/tests/deploy_target.rs

```diff
@@ -0,0 +1,337 @@
+use super::wrangler_toml::WranglerToml;
+
+use crate::settings::toml::deploy_target::Zoned;
+use crate::settings::toml::route::Route;
+use crate::settings::toml::DeployTarget;
+use crate::settings::toml::Manifest;
+
+// TOP LEVEL TESTS
+#[test]
+fn it_can_get_a_top_level_zoneless_deploy_target() {
+    let test_toml = WranglerToml::webpack_zoneless("zoneless", true);
```

Comment:
nit: separate `true` to `workers_dev = true`

Reply:
is in a later commit to a later pr. we talked about this in air